### PR TITLE
Us029 upload ci without ce

### PR DESCRIPTION
**Pull request by emilioward**
This change is part of us029 allowing a collection instrument to be uploaded without having a collection exercise

### CODE REVIEW

File: .pytest_cache/v/cache/lastfailed [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (tej99) [inline .pytest_cache/v/cache/lastfailed]: don't think you wanted to push this folder

File: application/controllers/collection_instrument.py [hunk 2]
```diff
@@ -99,6 +99,31 @@ def upload_instrument(self, exercise_id, file, ru_ref=None, classifiers=None, se
             raise RasError('Failed to publish upload message', 500)
         return instrument
 
+    @with_db_session
+    def upload_instrument_with_no_collection_exercise(self, survey_id, classifiers=None, session=None):
+        """
+        Encrypt and upload a collection instrument to the db
```
Comment (tej99) [inline application/controllers/collection_instrument.py]: This function does not encrypt just uploads.

File: application/views/collection_instrument_view.py [hunk 3]
```diff
@@ -7,14 +7,17 @@
...
 log = structlog.wrap_logger(logging.getLogger(__name__))
 
 collection_instrument_view = Blueprint('collection_instrument_view', __name__)
 
 COLLECTION_INSTRUMENT_NOT_FOUND = 'Collection instrument not found'
 NO_INSTRUMENT_FOR_EXERCISE = 'There are no collection instruments for that exercise id'
 UPLOAD_SUCCESSFUL = 'The upload was successful'
+UPLOAD_BAD_REQUEST = 'Upload bad request'
 LINK_SUCCESSFUL = 'Linked collection instrument to collection exercise'
+LINK_UNSUCCESSFUL = 'Unable to link collection instrument to collection exercise'
```
Comment (tej99) [inline application/views/collection_instrument_view.py]: don't think you are using LINK_UNSUCCESSFUL or UPLOAD_BAD_REQUEST anymore

**Comment by tej99** [comment]
[US029: INT Select/Load EQ Collection Instruments [XXL]](https://trello.com/c/QharabAR/42-us029-int-select-load-eq-collection-instruments-xxl)

File: tests/views/test_collection_instrument_view.py [hunk 4]
```diff
@@ -55,6 +55,20 @@ def test_collection_instrument_upload(self):
 
         self.assertEqual(len(collection_instruments()), 2)
 
+    def test_upload_collection_instrument_without_collection_exercise(self):
```
Comment (benjefferies) [inline tests/views/test_collection_instrument_view.py]: Probably want a test for creating a survey if one didn't exist

**Pull request merged**